BraveWallet/Crypto/BuySendSwap/SwapCryptoView.swift

```diff
@@ -362,16 +362,29 @@ struct SwapCryptoView: View {
     .listRowBackground(Color(.secondaryBraveGroupedBackground))
     Section(
       header:
-        Button(action: {
-          swapTokensStore.prepareSwap()
-        }) {
-          Text(swapButtonTitle)
+        VStack(spacing: 16) {
+          Text(
+            String.localizedStringWithFormat(Strings.Wallet.braveSwapFeeDisclaimer, {
+              let formatter = NumberFormatter()
+              formatter.numberStyle = .percent
+              formatter.minimumFractionDigits = 3
+              formatter.maximumFractionDigits = 3
+              return formatter.string(from: NSNumber(value: 0.00875)) ?? ""
```

Comment:
what is this 0.00875?

Reply:
The 0.875% constant Brave fee. I didn't want to include it in the translation copy in case it ever changes it'd be easier to adjust this number than get a new string re-translated across all languages

Reply:
i see. can we declare it as a property with the name explains this value?

Reply:
Done